Sources/CoreMetrics/SystemMetrics.swift

```diff
@@ -0,0 +1,322 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Metrics API open source project
+//
+// Copyright (c) 2018-2020 Apple Inc. and the Swift Metrics API project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift Metrics API project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Dispatch
+
+#if os(Linux)
+import Glibc
+#endif
+
+extension MetricsSystem {
+    /// `bootstrapSystemMetrics` is an one-time configuration function which globally enables system level metrics.
+    /// `bootstrapSystemMetrics` can be only called once, unless cancelled usung `cancelSystemMetrics`, calling it more
+    /// than once without cancelling will lead to undefined behaviour, most likely a crash.
+    /// There is no need to call `bootstrapSystemMetrics` if `SystemMetricsOptions` were provided to `bootstrap`.
+    ///
+    /// - parameters:
+    ///     - options: Options to configure `SystemMetricsHandler`
+    public static func bootstrapWithSystemMetrics(_ factory: MetricsFactory, options: SystemMetrics.Options) {
+        let factory = SystemMetricsFactory(factory: factory, options: options)
+        self.bootstrap(factory)
+        self.systemMetrics!.poll()
+    }
+    
+    /// Cancels the collection of system metrics. Calling this unlocks `bootstrapSystemMetrics` so it can be
+    /// called again.
+    public static func cancelSystemMetrics() {
```

Comment:
can you elaborate when you see this API being used

Reply:
I think this was leftover from a previous round of work. Will remove it 👍